index.html

```diff
@@ -5,8 +5,50 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Microverse Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+    <header>
+        <nav id="mobile-navbar">
+            <img src="assets/menu.png" alt="">
+        </nav>
+    </header>
+    <section id="background-image">
+        <h1 >
+            <span id="name-heading">
+                Hey there<br>
+            I'm Abdul Salam
+            </span><br>
+            <span id="job-role">
+                I',m a Software<br>
+                Developer
+            </span>
+            
+        </h1>        
+        <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p>
```

Comment:
Just as requested by the previous reviewer, let's try to ensure our project matches the given design as it is a requirement in this milestone😉. and it is also a good display of professionalism👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="320" alt="image" src="https://github.com/AbdulSalamAbro/portfolio-website/assets/64643995/b358817f-831b-4844-a18c-508dffc80154">|<img width="320" alt="image" src="https://github.com/AbdulSalamAbro/portfolio-website/assets/64643995/228fc268-b590-467d-a431-bd96e7d08347">|

Kindly take note of the positioning of each element in the design, the spacings (margins and padding), and the font properties, and try to make your project match the given design😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="375" alt="image" src="https://github.com/AbdulSalamAbro/portfolio-website/assets/64643995/2cff6441-f48f-40ea-b656-1da25b9f2dbd">|<img width="375" alt="image" src="https://github.com/AbdulSalamAbro/portfolio-website/assets/64643995/22d4505d-955b-4962-8170-4151c02ac300">|